### DISCUSSION TITLE
Is there any way to configure blocking and filtering of specific IPs that can access the teleport cluster?

### DISCUSSION THREAD

**Discussion by longansv** [gravitational/teleport#39643] [Q&A] [answered]
Is there any way to configure blocking and filtering of specific IPs that can access the teleport cluster? If so, please guide me how to configure it

**Comment by webvictim** [comment, marked as answer]
Not natively via Teleport. You can configure a firewall or WAF in front of a self-hosted Teleport server to achieve this.

**Reply by longansv** [reply]
yes, thank you very much
